boards/others/supermini_nrf52840/supermini_nrf52840_nrf52840_uf2_defconfig

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# Board Kconfig.defconfig enables USB CDC ACM and should disable USB remote
+# wakeup by default. It needs to be disabled here, because the USB nrfx
+# driver always overwrites option from Kconfig mentioned above with the
+# imply from CONFIG_USB_NRFX.
+CONFIG_USB_DEVICE_REMOTE_WAKEUP=n
+
+# Build UF2 by default, supported by the Adafruit nRF52 Bootloader
+CONFIG_BUILD_OUTPUT_UF2=y
+
+# UF2 Bootloader can use DT offset
+CONFIG_BOARD_HAS_NRF5_BOOTLOADER=n
+CONFIG_USE_DT_CODE_PARTITION=y
```

Comment:
you are setting this which means the top Kconfig has no use, but if you are doing this it means the application will be built for 0xc000, so how will the bootloader on the device be able to boot this? Presumably it is an nRF5 SDK bootloader which is looking for a softdevice at 0x1000 which will have an application at 0x26000 or 0x27000 (or other) depending upon softdevice version?

Reply:
Why would it be built for 0xc000? The dt partition is at 0x26000.
I left the kconfig to have the regular build which uses nordic/nrf52840_partition.dtsi, built as supermini_nrf52840/nrf52840, and the defconfig unsets that when building for uf2 with supermini_nrf52840/nrf52840/uf2.

Reply:
I missed that include, wasn't aware there was a uf2 partition file... Should be fine

Reply:
OK :+1: